docs/source/performance/caching.mdx

```diff
@@ -0,0 +1,225 @@
+---
+title: Caching
+sidebar_title: Caching
+---
+
+> Using cache control with Apollo Federation requires v0.28 of `@apollo/federation` in your subgraph, v0.36 of `@apollo/gateway` in your router, and v3.0.2 of Apollo Server in both servers. Please note that Apollo Server's cache hints API has evolved as of v3, so be sure to review the [updated caching documentation](https://www.apollographql.com/docs/apollo-server/performance/caching/).
+
+## Using cache hints with subgraphs
+
+To set [static cache hints](https://www.apollographql.com/docs/apollo-server/performance/caching/#in-your-schema-static) with Apollo Server-based subgraphs, the `@cacheControl` directive and `CacheControlScope` enum definitions must be included in the subgraph schema:
+
+```graphql
+enum CacheControlScope {
+  PUBLIC
+  PRIVATE
+}
+
+directive @cacheControl(
+  maxAge: Int
+  scope: CacheControlScope
+  inheritMaxAge: Boolean
+) on FIELD_DEFINITION | OBJECT | INTERFACE | UNION
+```
+
+The subgraph will calculate and set the cache hint for the response that it sends to the gateway and the gateway will then calculate the cache hint for the overall response based on the most restrictive settings among all of the responses received from the subgraphs involved in query plan execution.
```

Comment:
The docs use present tense in favor of future tense when they're equivalent in meaning:

"The subgraph will calculate" -> "The subgraph calculates and sets"

"that it sends to the gateway and the gateway will then calculate" -> "that it sends to the gateway, and the gateway then calculates"

Also recommend ending a sentence at "overall response" and starting a new sentence with "This hint is based on"